aii-opennebula/src/main/perl/opennebula.pm

```diff
@@ -472,20 +479,20 @@ sub stop_and_remove_one_vms
 sub remove_and_create_vm_template
```

Comment:
The name of the sub is confusing right now. It is more a remove_or_create_vm_template, depending on the $remove flag. I would make it more clear that what this sub does (removing or creating) is depending on the context it is called (configure or remove)